### PR TITLE
Migrate the VS project creation from godot to blazium

### PR DESCRIPTION
**Pull request by tokengamedev**
Primarily Renames the files generated, configuration and texts from godot to blazium.

### CODE REVIEW

**Review by RandomOfNoWhere — APPROVED** [review]

File: SConstruct [hunk 1]
```diff
@@ -340,24 +340,24 @@ if env["platform"] == "":
 
 if env["platform"] == "osx":
     # Deprecated alias kept for compatibility.
-    print_warning('Platform "osx" has been renamed to "macos" in Godot 4. Building for platform "macos".')
```
Comment (jss2a98aj) [inline SConstruct:343]: Blazium was not what changed the platform names, Godot 4 was accurate.

File: SConstruct [hunk 2]
```diff
@@ -774,7 +774,7 @@ else:
     # We apply it to CCFLAGS (both C and C++ code) in case it impacts C features.
     env.Prepend(CCFLAGS=["/std:c++17"])
 
-# Disable exception handling. Godot doesn't use exceptions anywhere, and this
+# Disable exception handling. Blazium doesn't use exceptions anywhere, and this
```
Comment (jss2a98aj) [inline SConstruct:777]: Please revert changes to comments, it makes merge conflicts much more likely in the future.

File: methods.py [hunk 3]
```diff
@@ -1564,9 +1564,10 @@ def generate_copyright_header(filename: str) -> str:
 /*  %s*/
 /**************************************************************************/
 /*                         This file is part of:                          */
-/*                             GODOT ENGINE                               */
```
Comment (jss2a98aj) [inline methods.py:1567]: Please do not change license notification headers, we have no gotten clarification on if doing so or not is correct yet.

File: methods.py [hunk 4]
```diff
@@ -1353,7 +1353,7 @@ def format_key_value(v):
                 vsconf = f'{target}|{a["platform"]}'
                 break
 
-        condition = "'$(GodotConfiguration)|$(GodotPlatform)'=='" + vsconf + "'"
```
Comment (jss2a98aj) [inline methods.py:1356]: This is a variable name, not the variable itself. Please do not change these so we can avoid merge conflicts. This goes for two other files as well.

File: methods.py [hunk 5]
```diff
@@ -10,7 +10,7 @@
 from pathlib import Path
 from typing import Generator, Optional
 
-# Get the "Godot" folder name ahead of time
```
Comment (jss2a98aj) [inline methods.py:13]: This file also has comments changed.

**Review by jss2a98aj — CHANGES_REQUESTED** [review]

File: SConstruct (hunk 1 above)
Comment (tokengamedev) [reply SConstruct:343]: will fix it in the squashed commit

File: SConstruct (hunk 2 above)
Comment (tokengamedev) [reply SConstruct:777]: will revert the non-required changes in comments in the squashed commit

File: methods.py (hunk 3 above)
Comment (tokengamedev) [reply methods.py:1567]: will revert this in squashed commit

File: methods.py (hunk 4 above)
Comment (tokengamedev) [reply methods.py:1356]: will revert all the variable names in the squashed commit

File: methods.py (hunk 5 above)
Comment (tokengamedev) [reply methods.py:13]: This comment is contextual and requires changing to blazium as there will be no Godot folder, other than that I will revert the rest of non-required comment and variable changes.
If you still insist I will change it.

Comment (jss2a98aj) [reply methods.py:13]: This one is fine then, thanks!

Comment (tokengamedev) [reply methods.py:13]: fixed

**Comment by tokengamedev** [comment]
The documentation needs to be updated too, but that will be a separate PR.

**Review by Bioblaze — APPROVED** [review]

**Review by Starkium — APPROVED** [review]

**Review by Norrox — APPROVED** [review]

**Review by jss2a98aj — APPROVED** [review]
The soloution builds with the correct name locally. I do not have Visual Studio installed to try building using it though.

**Comment by tokengamedev** [comment]
Even if you don't have VS installed you can create the solutions without building the whole project.

`scons platform=windows vsproj=yes dev_build=yes` 

It will create the solution, project and other files.

**Pull request merged**